includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -1554,6 +1558,20 @@ public function has_refund_failed( $order ) {
 		return 'failed' === $order->get_meta( '_wcpay_refund_status', true );
 	}
 
+	/**
+	 * Gets the payment method type used for an order, if any
+	 *
+	 * @param WC_Order $order The order to get the payment method type for.
+	 * @return string
+	 */
+	protected function get_payment_method_type_for_order( $order ) {
```

Comment:
I'd suggest making the method stricter: `private function get_payment_method_type_for_order( $order ): string`